articles/personal_growth.md

```diff
@@ -0,0 +1,33 @@
+
+Besides the soil a plant needs right at the beginning also some kind of pillar. As long as the roots are not deeply anchored, and the stem is still fragile it is important that the plant is supported. It is exactly like that with us trying to grow in our projects. By having all these challenges and opportunities to grow, sometimes we will make a mistake. Sometimes we are not fulfilling expectation of our stakeholders, maybe our estimation or planning was wrong due to wrong assumptions, we took wrong decisions or communicated not well or the way we acted was not as good as expected from our sponsor. These are the moments we need support and orientation when this situation takes us to our limits. We need someone taking care of us, helping that we will not break in the storm but become stronger and are prepared for the next time we face a similar situation.
+
+Most often we expect the project manager to support and stand up for us, but this is not always the case. This may be because of personal reasons or up to political topics like internal - external employee. From my perspective that doesn't matter if it is the PL or anyone else, as long as I know right from the beginning if I can count on this person or who is going to support me. This supporting person should be within the project stakeholders or at least within the area affected by my activities. It could be the PL, the sponsor, a team lead, someone from operation, business representatives and so on. Crucial is, that this person is aware of the project and what's going on, that I am linked and in close contact with the person, that I have an open and transparent communication to the person and that the person is a supporter of mine. For this reason, you should setup bilateral meetings or ask for feedback .
+
+Even if our terrain is nutritious and we have support, we still need water, heat and sunlight to grow. Only with this we can become what we shall or would like to be. Within our project that means that we have someone close to us who ensures that we deliver on time, within the expected quality and that we will not only choose the easy way but also try new solutions if time and place are right. We need someone who gives us the food required, meaning the tasks that match to us.
+
+Assuming that, this is the easiest way and what we will find most often within projects. People are asking for help or review, pair programming e.g. peer groups outside project to share experience and ask for advice. As this is most often related to objective questions or tasks, my experience is that most often people are happy to be linked with others to improve their working results. Most often in self-organizing teams team members switch tasks from one to each other to have best match of skills and performance for the respective task. Maybe you have also the possibility to be supported by a consulting coach or mentor.
```

Comment:
"my experience is that most often people are happy to be linked with others to improve their working results." -> das ist ein spannender Aspekt

"Maybe you have also the possibility to be supported by a consulting coach or mentor." -> auch hier eine Aussage die ich gerne weiter im Text sehen würde

Aber wie gesagt, Zeilen 22 und 24 passen irgendwie nicht recht rein